src/openrct2-ui/windows/Ride.cpp

```diff
@@ -337,6 +339,8 @@ static Widget _musicWidgets[] = {
     MakeWidget({  7, 47}, {302, 12}, WindowWidgetType::Checkbox, WindowColour::Secondary, STR_PLAY_MUSIC,     STR_SELECT_MUSIC_TIP      ),
     MakeWidget({  7, 62}, {302, 12}, WindowWidgetType::DropdownMenu, WindowColour::Secondary, STR_EMPTY                                     ),
     MakeWidget({297, 63}, { 11, 10}, WindowWidgetType::Button,   WindowColour::Secondary, STR_DROPDOWN_GLYPH, STR_SELECT_MUSIC_STYLE_TIP),
+    MakeWidget({154, 62}, {114, 114}, WindowWidgetType::FlatBtn,     WindowColour::Secondary),
+    MakeWidget({7, 90}, { 500, 450}, WindowWidgetType::Scroll,    WindowColour::Secondary,  SCROLL_BOTH  ),
     WIDGETS_END,
 };
```

Comment:
These were supposed to be aligned, but somehow someone messed up on line 340, can you fix it along with your additions?
Should look like:
```c++
static Widget _musicWidgets[] = {
    MAIN_RIDE_WIDGETS,
    MakeWidget({  7, 47}, {302,  12}, WindowWidgetType::Checkbox,     WindowColour::Secondary, STR_PLAY_MUSIC,     STR_SELECT_MUSIC_TIP      ),
    MakeWidget({  7, 62}, {302,  12}, WindowWidgetType::DropdownMenu, WindowColour::Secondary, STR_EMPTY                                     ),
    MakeWidget({297, 63}, { 11,  10}, WindowWidgetType::Button,       WindowColour::Secondary, STR_DROPDOWN_GLYPH, STR_SELECT_MUSIC_STYLE_TIP),
    MakeWidget({154, 62}, {114, 114}, WindowWidgetType::FlatBtn,      WindowColour::Secondary                                                ),
    MakeWidget({  7, 90}, {500, 450}, WindowWidgetType::Scroll,       WindowColour::Secondary, SCROLL_BOTH                                   ),
    WIDGETS_END,
};
```